data/json/furniture_and_terrain/terrain-floors-outdoors.json

```
@@ -146,16 +146,15 @@
     "connects_to": "DIRT",
     "move_cost": 3,
     "coverage": 40,
-    "flags": [ "TRANSPARENT", "DIGGABLE", "MOUNTABLE", "SUPPORTS_ROOF", "COLLAPSES", "INDOORS" ],
+    "flags": [ "DIGGABLE", "SUPPORTS_ROOF", "HIDE_PLACE" ],
```

I'm not sure these are big enough to block vision (100-200 dirt is 500-1000L, which theoretically is a 1x1x1m cube).
It's also not suitable as a HIDE_PLACE, it's a mound of dirt - you aren't actually going to be able to just crawl in - you might be able to make a shelter that you do so, but it takes time.

If it's a big mound of dirt, you should need to get rid of it before you can dig there, so it's probably not diggable.
There's no way a mound of dirt could support a roof.

```suggestion
    "flags": [ "TRANSPARENT" ],
```

---

As i had said, SUPPORTS_ROOF is mainly present for possible compatibility issues regarding its existence with current existing building spawns, although i suppose that is not as big of an issue as my exhausted stupor was worried it'd be

As for the diggable point, that makes sense, ~~although my main reason for keeping that one was for consistency with the other mound terrain tiles in the game, which are undiggable, despite having the diggable flag.~~ (Another case of me being to tired, such a flag was never present for the other mounds, unsure where i got that idea from. 

Hide place was iffy from the get-go, and definitely something i was already considering removing, so its nice to hear that my worry in that matter wasnt unjust.

And as for the Transparency, that also makes sense, i neglected to look at the values for the soil, and reference them with real life. 

Committing this suggestion ^^

---

By bad on missing that you mentioned in the PR description!